src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsPreview.java

```diff
@@ -0,0 +1,139 @@
+package com.wikia.webdriver.testcases.chattests;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate_TwoDrivers;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.chatpageobject.ChatPageObject;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+/**
+ * @author Rebekah Cunningham (modified from ChatTests to work on preview environment only)
+ * @description 1. Two staff users enter Chat 2. Verify switching between main and private
+ * message sections when one of the users has written public message 6. Verify switching
+ * between main and private message sections when one of the users has written private message
+ * 7. Verify non-staff users do not have permission to chat.
+ */
+public class ChatTestsPreview extends NewTestTemplate_TwoDrivers {
+
+  private Credentials credentials = config.getCredentials();
+  private String userStaff = credentials.userNameStaff;
+  private String userStaffPassword = credentials.passwordStaff;
+  private String userStaff2 = credentials.userNameStaff2;
+  private String userStaffPassword2 = credentials.passwordStaff2;
+  private String userOne = credentials.userName;
+  private String userOnePassword = credentials.password;
+  private String userTwo = credentials.userName2;
+  private String userTwoPassword = credentials.password2;
+
+
+  private static final int NUMBER_OF_PRIVATE_MESSAGES = 10;
+
+  private ChatPageObject openChatForUser(
+      WebDriver driver, String userName, String password
+  ) {
+    WikiBasePageObject base = new WikiBasePageObject(driver);
+    base.logInCookie(userName, password, wikiURL);
+    return base.openChat(wikiURL);
+  }
+
+
+  @Test(groups = {"ChatPreview_001", "ChatPreview"})
+  public void ChatPreview_001_twoUserEnterChat() {
+    switchToWindow(driverOne);
+    ChatPageObject chatUserOne = openChatForUser(
+        driverOne, userStaff, userStaffPassword
+    );
+    chatUserOne.verifyChatPage();
+
+    switchToWindow(driverTwo);
+    ChatPageObject chatUserTwo = openChatForUser(
+        driverTwo, userStaff2, userStaffPassword2
+    );
+    chatUserTwo.verifyChatPage();
+    switchToWindow(driverOne);
+
+    chatUserOne.verifyUserJoinToChatMessage(userStaff2);
+  }
+
+
+  @Test(groups = {"ChatPreview_002", "ChatPreview"})
+  public void Chat_002_verifySwitchingBetweenMainAndPrivateSections() {
```

Comment:
Method above is ChatPreview... and here Chat_... 
Please be more consistent and change it to ChatPreview here and in 003 method, or change method 001 to Chat_001_...